### ISSUE TITLE
Hyperparameter settings for WikiLSHTC experiments

### ISSUE THREAD

**Issue by ezshen** [bcol23/HyperIM#4]
Hi,

Thank you for your work. We are looking to replicate your numbers on the Wiki-LSHTC dataset. Can you post the hyperparameters needed to replicate your experiments?

**Comment by bcol23** [comment]
Thanks for your interest in our work. 

We use the hyperparameters recommended by the PoincaréGlove paper [Tifrea A, 2018] to pre-train the hyperbolic word embeddings. The embeddings size is set to 75*2D, which is a concatenation of 2D hyperbolic vectors.

We randomly initialize the label embeddings within (-0.001, 0.001) from a uniform distribution. The learning rate is set to 0.1. After 10 burn-in epochs with factor c = 10, they are trained for another 50 epochs. The embeddings size is consist with the word embeddings.

We then train HyperIM using RiemannianAdam with learning rate set to 0.0001 until converge. The size of hidden states in hyperbolic GRU is the same as the embeddings. The negative sampling number for partial interaction (training only) is set to 512.
